samples/bluetooth/iso_time_sync/include/iso_time_sync.h

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Not so sure about the location for the header file. Shouldn't it rather need to be placed to \ncs\nrf\include\bluetooth instead of samples/bluetooth/... ?

Reply:
This header is for this sample only